velox/functions/sparksql/String.h

```diff
@@ -75,9 +75,14 @@ template <typename T>
 struct AsciiFunction {
   VELOX_DEFINE_FUNCTION_TYPES(T);
 
-  FOLLY_ALWAYS_INLINE bool call(int32_t& result, const arg_type<Varchar>& s) {
-    result = s.empty() ? 0 : s.data()[0];
-    return true;
+  FOLLY_ALWAYS_INLINE void call(int32_t& result, const arg_type<Varchar>& s) {
+    if (s.empty()) {
```

Comment:
Would you update the documentation to clarify the semantics of these edge cases?